### DISCUSSION TITLE
is there any limit and palettes?

### DISCUSSION THREAD

**Discussion by CN1o2** [BEEmod/BEE2-items#3907] [General]
There is nothing to comment, the question is already in the title

**Comment by FrozenDragon0** [comment]
Well what do you mean by this exactly? Do you mean is there any limit to how many palettes can be used/increased?

**Reply by CN1o2** [reply]
Yes

**Reply by Konclan** [reply]
Read the [FAQ](https://github.com/BEEmod/BEE2.4/blob/master/FAQ.md), palette size is hard coded.

**Reply by saismeesaimonsaimes** [reply]
I think they meant how many palettes they can have saved in BEEmod.

**Comment by TeamSpen210** [comment]
There should be no limit, maybe it'd be annoying/slow if you had hundreds...
